tracing-subscriber/src/fmt/mod.rs

```diff
@@ -487,68 +354,92 @@ where
     /// [`timer`]: ./time/trait.FormatTime.html
     /// [`ChronoUtc`]: ./time/struct.ChronoUtc.html
     /// [`ChronoLocal`]: ./time/struct.ChronoLocal.html
-    pub fn with_timer<T2>(self, timer: T2) -> Builder<N, format::Format<L, T2>, F, W> {
-        Builder {
-            fmt_fields: self.fmt_fields,
-            fmt_event: self.fmt_event.with_timer(timer),
+    pub fn with_timer<T2>(self, timer: T2) -> SubscriberBuilder<N, format::Format<L, T2>, F, W> {
+        SubscriberBuilder {
             filter: self.filter,
-            settings: self.settings,
-            make_writer: self.make_writer,
+            inner: self.inner.with_timer(timer),
         }
     }
 
     /// Do not emit timestamps with log messages.
-    pub fn without_time(self) -> Builder<N, format::Format<L, ()>, F, W> {
-        Builder {
-            fmt_fields: self.fmt_fields,
-            fmt_event: self.fmt_event.without_time(),
+    pub fn without_time(self) -> SubscriberBuilder<N, format::Format<L, ()>, F, W> {
```

Comment:
Should this (and other) have refs to the inner builder methods?

Reply:
Yeah, I think that would be good (cc @davidbarsky)